src/utils.ts

```diff
@@ -288,4 +289,48 @@ export const totalSupply = async (): Promise<{ total: string, minted: string, pe
     }
 
     return { total: mintedBN.plus(pegKeepersBN).toString(), minted: mintedBN.toString(), pegKeepersDebt: pegKeepersBN.toString() };
-}
\ No newline at end of file
+}
+
+export const getLsdApy = memoize(async(name: 'wstETH' | 'sfrxETH'): Promise<{
+        apy: number,
+        baseApy: number,
+        apyMean30d: number,
+    }> => {
+    const response = await axios.get('https://yields.llama.fi/pools');
```

Comment:
This currently downloads 19346 pools with 13MB of data, we should never do that from the frontend